<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1400</width>
      <height>700</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionLog"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actionData_Show_View"/>
    <addaction name="actionReal_Time_Information"/>
    <addaction name="actionPos_Plot"/>
    <addaction name="actionAngle_Plot"/>
    <addaction name="actionSpeed_Plot"/>
    <addaction name="actionAltitude_Plot"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionInstructions"/>
    <addaction name="actionSupport"/>
    <addaction name="actionUpdates"/>
    <addaction name="actionAbout_Saic_Position_Manager"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave"/>
   <addaction name="actionLog"/>
   <addaction name="actionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionSuspend"/>
   <addaction name="actionStop"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionTile"/>
   <addaction name="actionCascade"/>
  </widget>
  <widget class="QDockWidget" name="Device_List">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>700</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>设备列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_Device_List">
    <widget class="QGroupBox" name="Serial_GroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>串口设置</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Lab_Serial_1">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>串口检测：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Check_Button">
        <property name="text">
         <string>检测串口</string>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Lab_Serial_2">
        <property name="text">
         <string>串口选择：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="Choice_ComboBox"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="State_Label">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Lab_Serial_3">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="Baudrate_ComboBox">
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>76800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Lab_Serial_4">
        <property name="text">
         <string>数据位：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="Bytesize_ComboBox">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Lab_Serial_5">
        <property name="text">
         <string>校验位：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="Parity_ComboBox">
        <item>
         <property name="text">
          <string>N</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Lab_Serial_6">
        <property name="text">
         <string>停止位：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="Stopbits_ComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="Open_Button">
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QPushButton" name="Close_Button">
        <property name="text">
         <string>关闭串口</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="Lab_Serial_7">
        <property name="text">
         <string>已接收：</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="Receive_LineEdit"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="Lab_Serial_8">
        <property name="text">
         <string>已发送：</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="Send_LineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>杆臂向量</string>
     </property>
     <widget class="QPushButton" name="Setting_Button">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>导入设置</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>28</y>
        <width>128</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_X">
         <property name="text">
          <string>X轴：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X_LineEdit">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>54</y>
        <width>128</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_Y">
         <property name="text">
          <string>Y轴：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Y_LineEdit">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>128</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_Z">
         <property name="text">
          <string>Z轴：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Z_LineEdit">
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>181</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>绘图设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>23</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="Start_Plot_Button">
         <property name="text">
          <string>开始绘图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Stop_Plot_Button">
         <property name="text">
          <string>停止绘图</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/07.png</normaloff>:/pic/images/07.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/10.png</normaloff>:/pic/images/10.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭菜单</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/01.png</normaloff>:/pic/images/01.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/09.png</normaloff>:/pic/images/09.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionMessage_Window">
   <property name="text">
    <string>Message Window</string>
   </property>
  </action>
  <action name="actionData_Show_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/14.png</normaloff>:/pic/images/14.png</iconset>
   </property>
   <property name="text">
    <string>数据收发栏</string>
   </property>
   <property name="toolTip">
    <string>数据收发栏</string>
   </property>
  </action>
  <action name="actionReal_Time_Information">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/15.png</normaloff>:/pic/images/15.png</iconset>
   </property>
   <property name="text">
    <string>实时信息栏</string>
   </property>
   <property name="toolTip">
    <string>实时信息栏</string>
   </property>
  </action>
  <action name="actionPos_Plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/16.png</normaloff>:/pic/images/16.png</iconset>
   </property>
   <property name="text">
    <string>经纬度绘图栏</string>
   </property>
   <property name="toolTip">
    <string>经纬度绘图栏</string>
   </property>
  </action>
  <action name="actionAngle_Plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/18.png</normaloff>:/pic/images/18.png</iconset>
   </property>
   <property name="text">
    <string>姿态角绘图栏</string>
   </property>
   <property name="toolTip">
    <string>姿态角绘图栏</string>
   </property>
  </action>
  <action name="actionSpeed_Plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/19.png</normaloff>:/pic/images/19.png</iconset>
   </property>
   <property name="text">
    <string>车速绘图栏</string>
   </property>
   <property name="toolTip">
    <string>车速绘图栏</string>
   </property>
  </action>
  <action name="actionAltitude_Plot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/17.png</normaloff>:/pic/images/17.png</iconset>
   </property>
   <property name="text">
    <string>高度绘图栏</string>
   </property>
   <property name="toolTip">
    <string>高度绘图栏</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/02.png</normaloff>:/pic/images/02.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/21.png</normaloff>:/pic/images/21.png</iconset>
   </property>
   <property name="text">
    <string>次级展开窗口</string>
   </property>
   <property name="toolTip">
    <string>次级展开窗口</string>
   </property>
  </action>
  <action name="actionInstructions">
   <property name="text">
    <string>指导手册</string>
   </property>
   <property name="toolTip">
    <string>指导手册</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>技术支持</string>
   </property>
   <property name="toolTip">
    <string>技术支持</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>软件更新</string>
   </property>
   <property name="toolTip">
    <string>软件更新</string>
   </property>
  </action>
  <action name="actionAbout_Saic_Position_Manager">
   <property name="text">
    <string>关于Saic Position Manager</string>
   </property>
   <property name="toolTip">
    <string>关于Saic Position Manager</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/08.png</normaloff>:/pic/images/08.png</iconset>
   </property>
   <property name="text">
    <string>记录数据</string>
   </property>
   <property name="toolTip">
    <string>记录数据</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/03.png</normaloff>:/pic/images/03.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionSuspend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/04.png</normaloff>:/pic/images/04.png</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/05.png</normaloff>:/pic/images/05.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/06.png</normaloff>:/pic/images/06.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="toolTip">
    <string>清除接收的数据</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/pic/images/20.png</normaloff>:/pic/images/20.png</iconset>
   </property>
   <property name="text">
    <string>平铺窗口</string>
   </property>
   <property name="toolTip">
    <string>平铺窗口</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="apprcc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionStart</sender>
   <signal>triggered()</signal>
   <receiver>Open_Button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>Close_Button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
